src/components/newNote/notes.css

```diff
@@ -0,0 +1,49 @@
+.input-notes-container {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+
+  position: relative;
+  width: 70%;
+  margin: 0 4rem 4rem 4rem;
+  box-shadow: 0 3px 5px rgb(43 43 43 / 21%);
+  padding: 2rem;
+}
+
+.btn-pin {
+  position: absolute;
+  top: 2.5rem;
+  right: 2rem;
+
+  background-color: transparent;
+  color: var(--SECONDARY-COLOR);
+  cursor: pointer;
+  z-index: 10;
```

Comment:
suggestion - you can use css variables for z-index which will help in debugging for issues related to z-index & it will be easier to visualize the stacking context